Notice.md

```diff
@@ -6,50 +6,99 @@ the open-source TEE maintained by Linaro, with initial contributions from
-	executes. It is distributed mostly under the BSD 2-clause open-source
-	license. It includes few external files under BSD 3-clause license or
-	other free software licenses.
--	The optee_linuxdriver git, containing the source code for the TEE driver
-	in Linux. This component implements a generic TEE driver, designed
-	primarily for TEE implementations that rely on the ARM
-	TrustZone(R)technology. It is distributed under the GPLv2 open-source
-	license. Please note that re-distribution under other versions of the
-	GPL license is not allowed. The rationale behind this limitation is to
-	ensure that this code may be used on products which have security
-	devices which prevent reloading the code. Such security devices would be
-	incompatible with some licenses such as GPLv3 and so distribution under
-	those licenses would be inconsistent with this goal. Therefore it is
-	recommended that care be taken before redistributing any of the
-	components under other license terms than those provided here.
+OP-TEE comes with several components:
+-	a secure privileged layer, executing at ARM secure PL-1 level,
+-	a set of secure userland libraries designed for Trusted Applications
+	needs,
+-	a Linux kernel driver merged since v4.12,
+-	a Linux userland library designed upon the GPD TEE Client API
+	specifications
+-	a Linux userland supplicant application for remote services expected by
+	the TEE OS,
+-	and some build scripts, debugging tools and examples to ease its
+	integration and the development of trusted applications and secure
+	services.
+
+These components are available from several git repositories. The main ones are
+the [optee_os], the [optee_client] and the [Linux kernel] since v4.12.
+
+The [optee_os] git contains the source code for the TEE OS itself. It includes
+the secure privileged layer hosting the Trusted Applications and libraries complying
+with the TEE Internal Core API v1.1. It is distributed mostly under the [BSD 2-Clause]
+open-source license. It includes few external files under [BSD 3-Clause] license or
+other free software licenses.
+
+The [optee_client] git contains the source code for the TEE client library in a
+Linux OS providing the TEE Client API v1.0. It is distributed under the
+BSD 2-clause open-source license.
+
+The [Linux kernel] containing the source code for the OP-TEE Linux driver. It is
+distributed under the GPLv2 open-source license.
+
+There are other OP-TEE components one might be interested in. The OP-TEE release tag
+references several git repositories enabling OP-TEE build and test for various
+platforms. Refer to the [build documentation] for information. The [optee_test] git
+repository proposes test materials through the `xtest` tool and dedicated trusted
+applications. The [optee_examples] git repository contains examples of TEE client
+and trusted applications and few helper documentation to get hand on trusted
+application development.
+
+Documentation
+------
+
+-	The [build documentation].
```

Comment:
Do we need an exhaustive list here? The problem with such a list is that it will need to be maintained. Perhaps it's enough with only a few of the most important ones needed to find the rest?

Reply:
+1 I would even say: just link to the documentation/ directory. The file names are supposed to be self-explanatory.

Reply:
agree. I'll fix